test/runtests.jl

```diff
@@ -102,6 +103,57 @@ end
     end
 end
 
+@testset "Lennard-Jones gas energy conservation" begin
```

Comment:
If this test times out CI you could put it in a new file `runtests_long.jl` and just test it locally. We don't want to limit our testing due to CI usage concerns, which will probably (hopefully!) become more of an issue as the package grows.

Reply:
I think that the tests could be restructured a bit. I was thinking we can group (in a separate PR) either by potentials or by functionality (i.e. energy conservation).